Doc/whatsnew/3.8.rst

```diff
@@ -1361,7 +1344,7 @@ Deprecated
   (Contributed by Victor Stinner in :issue:`37481`.)
 
 * Deprecated methods ``getchildren()`` and ``getiterator()`` in
-  the :mod:`~xml.etree.ElementTree` module emit now a
+  the :mod:`~xml.etree.ElementTree` module now emit a
```

Comment:
emits

Reply:
No -- 'methods' is the plural subject, so it's "methods X and Y now emit".